openBCIConstants.js

```diff
@@ -519,6 +530,58 @@ module.exports = {
     OBCISDLogForMin30:kOBCISDLogForMin30,
     OBCISDLogForSec14:kOBCISDLogForSec14,
     OBCISDLogStop:kOBCISDLogStop,
+    /** SD Card String Commands */
+    OBCIStringSDHour1:kOBCIStringSDHour1,
+    OBCIStringSDHour2:kOBCIStringSDHour2,
+    OBCIStringSDHour4:kOBCIStringSDHour4,
+    OBCIStringSDHour12:kOBCIStringSDHour12,
+    OBCIStringSDHour24:kOBCIStringSDHour24,
+    OBCIStringSDMin5:kOBCIStringSDMin5,
+    OBCIStringSDMin15:kOBCIStringSDMin15,
+    OBCIStringSDMin30:kOBCIStringSDMin30,
+    OBCIStringSDSec14:kOBCIStringSDSec14,
+    /**
+     * @description Converts a sd string into the proper setting. Defaults to 14 seconds.
```

Comment:
Actually it did not so good catch. This function returns an error if the `stringCommand` is not proper.